tests/all-tests.js

```diff
@@ -69,6 +69,8 @@ var testIncludes = [
     "../gpii/node_modules/ontologyHandler/test/node/OntologyHandlerTests.js",
     "../gpii/node_modules/pouchManager/test/pouchManagerTests.js",
     "../gpii/node_modules/preferencesServer/test/preferencesServerTests.js",
+    "../gpii/node_modules/preferencesServer/test/cloudSafeCredTests.js",
```

Comment:
Add preferencesServiceTests.js too.

Reply:
Oh yes, it would help if I added the test cases I had just converted!